tests/CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
 enable_testing()
 
 set (SOURCES
-     timer_test.cpp)
+     ../timer.cpp timer_test.cpp)
```

Comment:
I don't know of a better way at the moment. I think the tests should be connected to the project and then you don't need to specify a set of dependent sources for testing. I'll learn about how to do it later.